src/cascadia/TerminalSettingsModel/defaults.json

```diff
@@ -331,6 +331,8 @@
         { "command": { "action": "moveFocus", "direction": "left" }, "keys": "alt+left" },
         { "command": { "action": "moveFocus", "direction": "right" }, "keys": "alt+right" },
         { "command": { "action": "moveFocus", "direction": "up" }, "keys": "alt+up" },
+        { "command": { "action": "moveFocus", "direction": "previous" }, "keys": "ctrl+shift+left" },
+        { "command": { "action": "moveFocus", "direction": "next" }, "keys":  "ctrl+shift+right" },

```

Comment:
I feel like we're using `alt` as the button for "this has to do with panes". Maybe these should be `ctrl+alt+left/right`?